### DISCUSSION TITLE
Custome App For Revolt API

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#678] [Feature Suggestions]
CLOSED DISCUSSION PLEASE DELETE THREAD

**Comment by IAmTomahawkx** [comment]
It seems that you're suggesting app plug-ins, which are already something we support. Can you explain how this request differs from the plug-in support already offered?
Alternatively to plug-ins, third party clients already exist for revolt. You can find a list of them [here](https://github.com/revoltchat/awesome-revolt?tab=readme-ov-file#-third-party-clients)

**Reply by ghost** [reply]
We can tie this into customer apps which is like the customer bot feature so users can automatically add their own plugins and apps to their client. Gmail do this by allowing users create custom apps to manage their emails, cloud flare do this with service workers it's more complex than the already added plug in feture

**Reply by IAmTomahawkx** [reply]
Ah I see. This type of operation is a server-side action, which isn't something we could reasonably offer without significantly more funds (running arbitrary code for users requires a lot of both security measures and machine resources). An alternative to this is perhaps running a bot on your account to perform actions on your behalf?
